### ISSUE TITLE
'Change spelling' quick-fix should probably be the highest priority fix

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScript#23258]
```ts
class Foo {
    props

    rendar() {
        this.porps
    }
}
```

**Expected**: `Change spelling to 'props'` should be the first item in the list.
**Actual**:

![image](https://user-images.githubusercontent.com/972891/38473048-ff4a3140-3b3d-11e8-8aea-a6fa5c6b4e22.png)

**Comment by krryan** [comment]
I agree that it should have priority over declaring a new property (named or index signature).

But it definitely should not have priority over importing something that the name matches. I have seen 'change spelling' as an option in situations where I'm referencing some existing export, looking to use the 'import' quick-fix. That should have still-higher priority; changing the spelling to some similarly-named deal is not what is likely wanted in the situation where the "spelling error" matches some other export.
